booking-movie/src/components/Booking/BookingPopUp.js

```diff
@@ -0,0 +1,114 @@
+import moment from "moment";
+import React, { useEffect } from "react";
+import { Form, Row, Button } from "react-bootstrap";
+import $ from "jquery";
+import CryptoJS from "crypto-js";
+import { useDispatch } from "react-redux";
+import { getMovieShowtimesAction } from "../../redux/actions/movieActions";
+import { Link } from "react-router-dom";
+
+function BookingPopUp(props) {
+    const { movieId, data } = props;
+    const dispatch = useDispatch();
+
+    var timeFrom = (x) => {
+        var dates = [];
+        for (let i = 0; i < Math.abs(x); i++) {
+            const day = new Date(
+                new Date().getTime() -
+                    (x >= 0 ? i : i - i - i) * 24 * 60 * 60 * 1000
```

Comment:
```suggestion
                    (x >= 0 ? i : -i) * 24 * 60 * 60 * 1000
```
Đồng thời tách riêng phần này ra thành 1 biến thay vì gộp chung 1 dòng như này